### ISSUE TITLE
Mushop contact page to accept Send a Note feedback input and store the data in ADB

### ISSUE THREAD

**Issue by karthiksuku** [oracle-quickstart/oci-cloudnative#284]
Mushop Contact page to accept "Send a Note" feedback input and store the data in ADB

**Comment by jeevanjoseph** [comment]
This is a good suggestion. However there are multiple services that interact with ADB today. Orders uses ATP, carts uses the JSON collection storage in ATP, catalogue uses it for product info, users etc. 

Is there a specific DB feature that we can showcase with this enhancement?

**Comment by karthiksuku** [comment]
Hi Jeevan ,  currently I have used used Order in ADB to trigger intergation to backend EBS / Fusion ERP using OIC. 

Regarding the specifics with this reqest - What I am wanting is a way to capture feedback from the comments page - and it gets stored in ADB . Once its there - I am tyring to create APEX app and enbedd AI Language service to analyse the sentiment and display customer sentiment analysis APEX dashboard.

**Comment by jeevanjoseph** [comment]
That’s an interesting one. Not sure why a DB or apex is required though. The Language service is seemingly is available over REST, and one could simply call the endpoint with text to analyze it, and display results right on the UI.  If at all, perhaps an ephemeral store/cache like redis might suffice. 

If you wanted to build a standalone apex app to showcase language service, you could simply build a form for text entry in apex itself.
 
The way I see this making sense for MuShop is to build a new service for sentiment analysis and use that service to call language service. However the credentials management needs serious thought.

**Comment by karthiksuku** [comment]
well summarised re sentiment analysis. At this stage the reqest is simple - Mushop to enable rudumentaty function to capture feedback @ Mushop Contact page and store in ADB.

**Comment by jeevanjoseph** [comment]
The rudimentary persistence of feedback items to an ADB instance does not add anything new to MuShop, as we have multiple examples of persisting data into ADB with various languages and frameworks. Of Course, you're free to implement it for your needs if you choose. I don't think I'd favor a PR on that to the main branch at this time. @mvandervliet & @junior - what's your opinion ? 

Sentiment analysis as a separate feature is pretty interesting and we can track that using a new issue.

**Comment by karthiksuku** [comment]
Hi Jeevan , lets zoom to discuss. To use the sentiment analysis - we need to persist the feedback data somewhere... if it's in ADB it's easier for users to consume and innovate

**Comment by mvandervliet** [comment]
Yes, warrants discussion. Generally speaking eCommerce apps always include features for reviews/ratings/feedback. In MuShop, this is omitted for sake of feature simplification where the concept is deliberately broad for the purpose of demonstration. I'm not opposed to accommodating this feature/service(s) with the assumption that administrative/inspection would be done via extensions (like apex) as a separate showcase. Any such demos or integrations should be also be documented and shareable.

**Comment by karthiksuku** [comment]
Thank Matt / Jeevan
